doc/option_drop_group.md

```diff
@@ -0,0 +1,97 @@
+# Option Drop Group Database
+
+## Description 
+Explanation of the `db/option_drop_groups.conf` file and structure.
+
+This database file allows the creation of groups of random options
+that will be added to certain equipments when dropped. After creating
+a group in this database file, you may set up drops in `mob_db` to use
+it in order to get items with these options. For more information on
+adding option drop groups to `mob_db`, check `doc/mob_db.txt` documentation file.
+
+Each item may have up to `MAX_ITEM_OPTION` options at the same time,
+in this document, each of these independent options will be called
+`option slot`. One drop group will define the possibilities of random
+options for each of these slots.
+
+## Entries Format
+
+```
+<Group Name Constant>: (
+	{ // Option Slot 1
+		Rate: (int) chance of filling option slot 1 (100 = 1%)
+
+		// Possible options for slot 1
+		// min/max value : int, defaults to 0
+		// chance : int, 100 = 1% if not set, will be 100%/number of possibiltiies
+		OptionName: value
+		// or
+		OptionName: [min value, max value]
+		// or
+		OptionName: [min value, max value, chance]
+		// ...  (as many as you want)
+	},
+	// ... (up to MAX_ITEM_OPTION)
+),
+```
+
+### `Group Name Constant`
+This is the group name, it is how this group is referenced in other files
+(e.g. mob_db). It must be globally unique, as it is a server constant, and
+must contain only letters, numbers and "_".
```

Comment:
Github Markdown error
underscore _ will change text italic
just adding a space will remove the italic font